src/main/java/com/exactpro/th2/inframgr/k8s/Kubernetes.java

```diff
@@ -456,15 +468,15 @@ public void startInformers() {
         informerFactory.startAllRegisteredInformers();
     }
 
-    private KubernetesClient client;
+    private final KubernetesClient client;
 
     private Map<String, MixedOperation> operations;
 
     private String namespace;
 
-    public Kubernetes(K8sConfig config, String schemaName) {
-
-        // if we are not using custom configutation, let fabric8 handle initialization
+    public Kubernetes(BehaviourCfg behaviour, K8sConfig config, String schemaName) {
+        this.permittedToRemoveNamespace = behaviour.isPermittedToRemoveNamespace();
```

Comment:
`behaviour` can be null. We should add explicit check to throw correct exception in this case or handle null value